iOS/DuckDuckGo/TabsModelPersistence.swift

```diff
@@ -47,7 +47,7 @@ class TabsModelPersistence: TabsModelPersisting {
     convenience init() throws {
 
         guard let appSupportDir = FileManager.default.urls(for: .applicationSupportDirectory, in: .userDomainMask).first else {
-            Pixel.fire(pixel: .tabsStoreSupportDirAccessError)
+            DailyPixel.fireDailyAndCount(pixel: .tabsStoreSupportDirAccessError)
```

Comment:
Won't plain transition to dailyAndCount change the name of the pixel from regualr to the one with count appended? I'd prefer to leave the old pixel as is, just add the daily one - I think there's an option for that in Daily Pixel API.